frontend/src/app/application-forms/fields/number-of-cords.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-container [formGroup]="applicantInfo">
+    <label class="usa-input" for='number-of-cords' id="email-label">Number of Cords<span class="required-fields-asterisk">*</span></label>
```

Comment:
Should the `id` on this be changed to `number-of-cords-label`? I can never remember how `aria-labelledby` works so maybe it's fine, but I guess then you might end up with duplicate DOM IDs and that might get wonky.